docs/run_test/platform.rst

```diff
@@ -169,6 +172,15 @@ deployment.
   public IP addresses.  False means to connect with the private IP addresses.
   If not provided, the connections will default to using the public IP
   addresses.
+* **ignored_capability**. Specify feature names which will be ignored in 
+  test requirement. You can find the feature name in files under ``lisa/features``.
+  For example, ``lisa/features/isolated_resource.py`` defined feature name as below:
+
+   .. code:: python
+
+         FEATURE_NAME_ISOLATED_RESOURCE = "isolated_resource"
```

Comment:
It needs to tell users the source to look for value. The value is from the name method of a feature.

```
    @classmethod
    def name(cls) -> str:
        return FEATURE_NAME_ISOLATED_RESOURCE
```